content/03.methods.md

```diff
@@ -11,93 +11,115 @@ The prompt for the Methods section includes the formatting of equations with ide
-It utilizes version control and a continuous integration workflow to facilitate efficient and transparent collaboration among authors.
+The AI-based revision task is based on a machine learning model trained on a corpus of scientific papers from the same field as the manuscript (Figure {@fig:ai_revision}b).
+This model is used to identify and suggest revisions to the text.
+The output of the revision task is a modified version of the manuscript that is ready for submission.
+
+We implemented an AI-based revision infrastructure in Manubot [@doi:10.1371/journal.pcbi.1007128], a tool for collaborative writing of scientific manuscripts.
 Manubot integrates with popular version control platforms such as GitHub, allowing authors to easily track changes and collaborate on writing in real time.
-Additionally, Manubot automates the process of generating a formatted manuscript (such as HTML, PDF, DOCX; Figure {@fig:ai_revision}a shows the HTML output), reducing the time and effort required for manuscript preparation and submission.
+Furthermore, Manubot automates the process of generating a formatted manuscript (such as HTML, PDF, DOCX; Figure {@fig:ai_revision}a).
 Built on this modern and open paradigm, our AI-based revision software was built using GitHub Actions, which allows the user to easily trigger an automated revision task on the entire manuscript or specific sections of it.
+The AI-based revision task is based on a machine learning model trained on a corpus of scientific papers from the same field as the manuscript (Figure {@fig:ai_revision}b).
+This model is used to identify and suggest revisions to the text, resulting in a modified version of the manuscript that is ready for submission.
+This reduces the time and effort required for manuscript preparation and submission.
 
 
-When the user triggers the action, the manuscript is parsed by section and then by paragraph (Figure {@fig:ai_revision}b), passed to the language model along with a set of custom prompts, returned, reformatted, and output.
+When the user triggers the action, the manuscript is parsed by section and then by paragraph (Figure {@fig:ai_revision}b) and passed to the language model along with a set of custom prompts.
+The model then returns a reformatted version of the text, which is output.
 Our workflow then uses the GitHub API to generate a new pull request, allowing the user to review and modify the output before merging the changes into the manuscript.
-This workflow attributes text to either the human user or to the AI language model, which may be important if future legal decisions alter the copyright landscape around the outputs of generative models.
+This workflow attributes text to either the human user or to the AI language model, which may be important in light of potential future legal decisions that alter the copyright landscape around the outputs of generative models.
+
 
+We used the following equation to predict the cost of a run:
```

Comment:
It's making stuff up again

Reply:
Yeah, since this doesn't happen in other sections, it could be related to the "keep technical details" part in the prompt. When writing the prompt, I meant "keep technical details if they exist, but do not make them up" 😆

Reply:
Correction: it also happens in the Results section. It adds references to nonexisting Figures and Tables.